concepts/eventing/performance evaluation/nats-jetstream/different-cloud-providers/results.md

```diff
@@ -0,0 +1,113 @@
+- F3  - NATS JetStream just doesn’t accept and dispatch events despite storage is not full. This is solely observed on Azure
+<br/><br/>
+
+## Tests Results
+In this section the test outcomes of the 2 day running test with the [K6](https://k6.io/). As you see the given K6 tool rps (request-per-second) is different at what is observed at EPP. Therefore, the actual value is EPP rps.
+As you see GCP and AWS performed similarly, also SSD and HDD difference was not much. HDD cannot be tested for AWS as it doesn't offer such storage type.
+### GCP
+CPU: 4, Memory: 15GB
+| cluster | K6 rps | EPP rps | JS Crashed | version | test duration | Failure type | NATS version |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| gcp1 | 500 | 320 | yes | 2.7.0-rc2 | 5 hours | full storage (F1) |  |
+| gcp2 | 250 | 232 | no | 2.7.0-rc2 |  |  |  |
+| gcp3 | 350 | 315 | yes | 2.7.0-rc2 | 5.5 hours | full storage (F2) |  |
+| gcp4 | 300 | 263 | yes | 2.7.0-rc2 | 13.5 hours | full storage (F2) |  |
+| gcp5 | 280 | 240 | yes | 2.7.0-rc2 | 31.5 hours |  |  |
+| gcp6 | 280 | 265 | yes | main |  | full storage (F2) | 2.9.0 |
+| gcphdd1 | 250 | 240 | no | 2.7.0-rc2 |  |  |  |
+| gcp-hdd | 300 | 278 | yes | main |  | full storage (F1) | 2.9.0 |
+<br/>
+
+### Azure
+### Azure Performance Tests
+
+CPU: 4, Memory: 16GB
+
+| cluster | K6 rps | EPP rps | JS crashed | version | test duration | Failure type | NATS version |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| az12 | 250 | 88 (250 for 3h) | yes | main | 19.5 hours | F1 | 2.9.0 |
+| az13 | 300 | 93 (298 for 3h) |  | main |  |  | 2.9.0 |
+| az14 (hdd) | 300 | 70 (290 for 3h) |  |  |  |  | 2.9.0 |
+
+<br/>
+
+CPU: 2, Memory: 16GB
+
+| cluster | K6 rps | EPP rps | JS crashed | version | test duration | Failure type | NATS version |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| ssd | 350 | 88 | no | 2.7.0-rc2 |  |  |  |
+| ssd | 2400 | 140 | no | 2.7.0-rc2 |  |  |  |
+| ssd | 2400 | 84 | yes | main | 11 hours | full storage (F1) | 2.9.0 |
+| ssd | 4800 | 84 | yes | main |  | full storage (F2) | 2.9.0 |
+| ssd | 6000 | 174 | yes | 2.7.1 | 11.4 hours | full storage (F2) |  |
+| hdd | 1200 | 116 | yes | 2.7.0-rc2 | 4h 20 min | JS pod restart |  |
+| hdd | 4800 | 104 | yes | main | 6 hours | F3 - appears w/ new NATS version. Started sending events after consumer leader reelection | 2.9.0 |
+| hdd | 4800 | 180 | yes | main | 1.15 hours | F3 - appears w/ new NATS version. Started sending events after consumer leader reelection and restart of k6 pods.                      Probably full storage caused this, but nothing in the dashboard. | 2.9.0 |
+
+<br/>
+
+### AWS
+CPU: 4, Memory: 16GB
+|  | K6 rps | EPP rps | JS Crashed | version | test duration | Failure type  |
+| --- | --- | --- | --- | --- | --- | --- |
+| aws1 | 250 | 247 | no | 2.7.1 |  |  |
+| aws2 | 500 | 365 | yes | 2.7.1 | 5 hours | F1   -  storage is full |
+| aws3 | 350 | 307 | yes | 2.7.1 | 17 hours | F1   -   storage is full |
```

Comment:
For consistency, please let's fill-in the missing items per test execution (if you still have this info).